### PR TITLE
Update doc 0.9.0 release

### PR DESCRIPTION
**Pull request by houshengbo**

### CODE REVIEW

File: releases/0.9.0-incubating/INSTALL.md [hunk 1]
```diff
@@ -36,60 +36,65 @@ You need to install gpg on your local machine.
...
 
 ## Prerequisites
 
 If you are a Ubuntu user, our suggested version is between 14.04 and 16.04. Open a terminal, go to the directory of "incubator-openwhisk",
 and run the script "all.sh" under tools/ubuntu-setup:
 
 ```
-$ cd incubator-openwhisk-<version>
-$ ./tools/ubuntu-setup/all.sh
+cd incubator-openwhisk-<version>;
```
Comment (rabbah) [inline releases/0.9.0-incubating/INSTALL.md]: Remove ;

Comment (rabbah) [reply releases/0.9.0-incubating/INSTALL.md]: Is there version name not known to make the cd command complete?

File: releases/0.9.0-incubating/INSTALL.md [hunk 2]
```diff
@@ -139,18 +144,21 @@ remains clueless, please [log an issue](https://github.com/apache/incubator-open
 
 ## Deploy OpenWhisk
 
-Stay under the directory of incubator-openwhisk-<version>, and run the following ansible scripts one by one:
+We recommend you set the environment variable $OPENWHISK_HOME on your local machine to the extracted directory incubator-openwhisk-<version>.
+The deployment of OpenWhisk can also work without setting this environment variable. It is UNACCEPTABLE to set $OPENWHISK_HOME
+to empty or a wrong directory.
+
+Make sure your terminal goes to the directory of incubator-openwhisk-<version>/ansible, and run the following ansible
+scripts one by one:
```
Comment (rabbah) [inline releases/0.9.0-incubating/INSTALL.md]: If you’re going to use OPENWHISK_HOME as below for wsk property config then I suggest also 

cd $OPENWHSIK_HOME/ansible

File: releases/0.9.0-incubating/INSTALL.md (hunk 1 above)
Comment (rabbah) [reply releases/0.9.0-incubating/INSTALL.md]: I suggest moving OPENWHISK_HOME paragraph here and using the property for the cd command as well.

**Comment by rabbah** [comment]
Thanks for making these improvements.

File: releases/0.9.0-incubating/INSTALL.md (hunk 1 above)
Comment (houshengbo) [reply releases/0.9.0-incubating/INSTALL.md]: done.

File: releases/0.9.0-incubating/INSTALL.md (hunk 2 above)
Comment (houshengbo) [reply releases/0.9.0-incubating/INSTALL.md]: done

**Comment by rabbah** [comment]
I pushed edits to address my comments directly. Thanks @houshengbo.

**Pull request merged**